app/src/sharedTest/java/org/oppia/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -105,6 +105,9 @@ class TopicRevisionFragmentTest {
     }
   }
 
+  //Test Failing in other devices too. RecyclerView item count
+  //Expected: <2>
+  //Got: <12>
   @Test
   fun testTopicRevisionFragment_loadFragment_checkSpanCoun_isTwo() {
```

Comment:
This test is failing in other devices too. There are 12 items in the recyclerview but check was made for 2 item count only.